test/SOME FILE.cpp

```diff
@@ -0,0 +1 @@
+

```

Comment:
Just to make sure readers do not get _that_ confused. 🙂
```suggestion
// Test file to make sure the filename formatter works as excepted

```